### PR TITLE
Porfolio Homepage

### PR DESCRIPTION
**Pull request by UsamaMasood12**
This project includes the use of Linters, Git, GitHub, Figma, and Visual Studio Code. In this project, I build a basic homepage for my personal portfolio. This portfolio is according to the mobile-first approach.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="box box1">Usama Masood</div>
+            <div class="box box2"><img src="Enabled.png" alt=""></div>
+            
+        </div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] This should be the toolbar of your page and you should use proper HTML tags to wrap the elements for better semantics and increase the SEO of your page. Kindly wrap this section in a `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

 - [ ] The logo should be on the left and has a link (`a` tag). Kindly fix that. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

 - [ ] The hamburger button is on the right and has a `nav` tag around it. The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices. Kindly change that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <div class="box box4">I'am a Software Developer</div>
+            
+        </div>
+        <div class="container2">
+            <div class="container3">
+                <div class="box box5">I can help you build a product , feature or<br>website Look through some of my work<br>and experience! If you like what you see<br>and have a project you need coded, don’t<br>hestiate to contact me</div>
+                <div class="box box6"><img src="Social Media.png" alt=""></div>
+            </div>
+            <div class="box box7"><img src="Illustration.png" alt=""></div>
+        </div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Kindly wrap this section in a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. The container of the section is a section tag.

 - [ ] The main title should be wrapped with an `h1` tag. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly fix that.

 - [ ] The subtitle uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. This way will be semantically correct and the screen readers and other technologies will thank you. Kindly wrap the subtitle following the `h` tag hierarchy and double-check for typos. 👍 
 
- [ ] The paragraph should be wrapped with a `p` tag. This way will be semantically correct and the screen readers and other technologies will thank you. Kindly change that.

- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Kindly remove that image and you could copy those icons from the Figma template and add them as individual images in the `a` tag inside the list so you can add custom links for your social pages to them.

- [ ]  You added the background image as an HTML image element. If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. Kindly remove the image and use the CSS background-image property to add it.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @UsamaMasood12, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Linters correctly configured and passing. ✔️
 - Nice PR title and description. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] You used correctly the Microverse template of the README file, but it is just a copy and you didn't add any information about this specific project or yourself. Kindly change the title and the data in the headers of the README file to make that customized to this project.
- [ ] The overall style of your page is mismatched with the mobile version of the Figma template. Since you are developing in a mobile first approach as you mentioned, you should apply the styles you find in the mobile version of the Figma template. It is possible to click on the elements to inspect and see the CSS properties used in the template page. Kindly change the sizing, font-sizes, font-family and some colors to match with the template.
- Your design:
- 
image
- Figma template:
- 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (leonardodiasb) [inline index.html]: - [ ] This should be the toolbar of your page and you should use proper HTML tags to wrap the elements for better semantics and increase the SEO of your page. Kindly wrap this section in a `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

 - [ ] The logo should be on the left and has a link (`a` tag). Kindly fix that. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

 - [ ] The hamburger button is on the right and has a `nav` tag around it. The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices. Kindly change that.

File: index.html (hunk 2 above)
Comment (leonardodiasb) [inline index.html]: - [ ] Kindly wrap this section in a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. The container of the section is a section tag.

 - [ ] The main title should be wrapped with an `h1` tag. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly fix that.

 - [ ] The subtitle uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. This way will be semantically correct and the screen readers and other technologies will thank you. Kindly wrap the subtitle following the `h` tag hierarchy and double-check for typos. 👍 
 
- [ ] The paragraph should be wrapped with a `p` tag. This way will be semantically correct and the screen readers and other technologies will thank you. Kindly change that.

- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Kindly remove that image and you could copy those icons from the Figma template and add them as individual images in the `a` tag inside the list so you can add custom links for your social pages to them.

- [ ]  You added the background image as an HTML image element. If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. Kindly remove the image and use the CSS background-image property to add it.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @UsamaMasood12, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Linters correctly configured and passing. ✔️
 - Nice PR title and description. ✔️
 - Professional README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] You correctly added your information and information about this project to the README and that's great. But you didn't add any of the changes related to the code of your project and your page, kindly add them to move to the next project. The overall style of your page is mismatched with the mobile version of the Figma template. Since you are developing in a mobile-first approach as you mentioned, you should apply the styles you find in the mobile version of the Figma template. It is possible to click on the elements to inspect and see the CSS properties used in the template page. Kindly change the sizing, font-sizes, font-family and some colors to match with the template.
- Your design:
- 
image
- Figma template:
- 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                            <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./images/Group.png" alt=""></a></li>
+                            <li><a href="#"><img src="./images/1.png" alt=""></a></li>
+                            <li><a href="https://twitter.com/Usama____Masood"><img src="./images/3.png" alt=""></a></li>
+                            <li><a href="#"><img src="./images/2.png" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="container2">
+                    <div class="box box1"><img src="./images/4.png" alt=""></div>
+                </div>
```
Comment (OyePriscilla) [inline index.html:57]: - [optional] Kindly please remove this image from here and use CSS background-image property to do that. * If an image is not part of the content it is always better to use CSS background-image property. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12  😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

- [ ] Kudos for working on your Readme file, but you need to work more on it and ensure that all the necessary details are customized. This will help non-professional to quickly understand what your project is all about and how to access your repo. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Debas-31 — APPROVED** [review]
> ## Hi @UsamaMasood12   :wave:, 

# Status: Approved :heavy_check_mark:  :heavy_check_mark:  :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Design template usage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**